lib/engine/game/g_1856.rb

```diff
@@ -309,6 +311,33 @@ def cert_limit
         POST_NATIONALIZATION_CERT_LIMIT[num_corporations][@players.size]
       end
 
+      def destination_check
```

Comment:
might as well put a ! here because it calls a destinate!

Reply:
Ok, it's changed now